posthog/api/notebook.py

```diff
@@ -241,17 +241,19 @@ def _filter_request(self, request: request.Request, queryset: QuerySet) -> Query
                 # each of those objects can have attrs which is a dict with id for the recording
                 for match_pair in match_pairs:
                     target, match = match_pair.split(":")
```

Comment:
Will this break if you send `contains=recordings` ? 

I'm thinking purely from an API point of view it might make sense to support:
1. `=recording` -> notebook contains at least one recording
2. `=recording:true` -> same as above
3. `=recording:false` -> notebook does not contain a recording
4. `=recording:123` -> notebook does contains recording with 123
5. `=recording,insight:ABC` -> notebook contains recording AND insight with id ABC